components/tooltip/index.css

```diff
@@ -122,6 +119,7 @@ governing permissions and limitations under the License.
   }
 
   &.is-open {
+    --spectrum-overlay-animation-distance: var(--mod-tooltip-animation-distance, var(--spectrum-tooltip-animation-distance));
```

Comment:
Overwriting this value in the `.is-open` state means that JS application will not have access to this value while the Tooltip is closed. I can't remember right off why someone was asking about that recently, but would it be that different to move this onto the default rule?

Reply:
I either didn't send this earlier or it disappeared, so for posterity:

Good catch! We don't need this overwrite since we've [refactored the transforms](https://github.com/adobe/spectrum-css/pull/2266/commits/51f31f5a703c9dd75c283cec35ceb239bc380604) so I removed it.